include/golioth/config.h

```diff
@@ -61,6 +61,18 @@
 #define CONFIG_GOLIOTH_OTA_MAX_VERSION_LEN 64
 #endif
 
+#ifndef CONFIG_GOLIOTH_OTA_MAX_COMPONENT_HASH_LEN
+#define CONFIG_GOLIOTH_OTA_MAX_COMPONENT_HASH_LEN 64
```

Comment:
Let's make the hash and bootloader name lengths constants in ota.h, rather than configurable values.